R/bug-report.R

```diff
@@ -0,0 +1,44 @@
+
+#' File an RStudio Bug Report
+#'
+#' A utility function to assist with the filing of an RStudio bug report.
+#' This function will pre-populate a template with information useful in
+#' understanding your reported bug.
+#'
+#'
+#' @export
+bugReport <- function() {
+
+  verifyAvailable()
+
+  rstudioInfo <- versionInfo()
+  rstudioEdition <- if (rstudioInfo$mode == "desktop") "Desktop" else "Server"
+  rstudioVersion <- format(rstudioInfo$version)
+
+  rInfo <- utils::sessionInfo()
+  rVersion <- rInfo$R.version$version.string
+  rVersion <- sub("^R version", "", rVersion, fixed = TRUE)
+  osVersion <- rInfo$running
+
+  templateFile <- system.file("resources/bug-report.md", package = "rstudioapi")
+  template <- readLines(templateFile)
+  rendered <- renderTemplate(template, list(
+    RSTUDIO_VERSION = rstudioVersion,
+    RSTUDIO_EDITION = rstudioEdition,
+    OS_VERSION      = osVersion,
+    R_VERSION       = rVersion
+  ))
+
+  if (requireNamespace("clipr", quietly = TRUE)) {
```

Comment:
This is not going to work with RStudio Server, so if we detect server mode I think we need to open it in a buffer and ask them to copy/paste themselves (or something similar).